CredentialProvider/Presenter/CredentialWelcomePresenter.swift

```diff
@@ -14,6 +15,16 @@ class CredentialWelcomePresenter: BaseWelcomePresenter {
     }
     
     private let credentialProviderStore: CredentialProviderStore
+
+    private var okButtonObserver: AnyObserver<Void> {
+        return Binder(self) { target, _ in
+            if let url = URL(string: "lockbox://fxa") {
```

Comment:
This is not a secure way to do universal linking; someone could register for this domain and phish FxA logins.

Reply:
Rather than deeplinking here, let's just dispatch the configured action and let users take the action on their own.